install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -692,6 +693,9 @@ data:
 {{- end }}
 {{- if hasKey .Values.nodePort "enableHealthCheck" }}
   enable-health-check-nodeport: {{ .Values.nodePort.enableHealthCheck | quote}}
+{{- end }}
+{{- if hasKey .Values.nodePort "enableHealthCheckLoadBalancerIP" }}
+  enable-health-check-loadbalancer-ip: {{ .Values.nodePort.enableHealthCheckLoadBalancerIP | quote}}
```

IMO it would be cleaner to have all value logic tied to `enable-health-check-loadbalancer-ip` here, like e.g for `install-no-conntrack-iptables-rules`

https://github.com/cilium/cilium/blob/8e37f1fc845c8148bae77df3c68751111b813f8a/install/kubernetes/cilium/templates/cilium-configmap.yaml#L595-L600

```suggestion
{{- end }}
{{- if .Values.gke.enabled }}
  enable-health-check-loadbalancer-ip: "true"
{{- else if hasKey .Values.nodePort "enableHealthCheckLoadBalancerIP" }}
  enable-health-check-loadbalancer-ip: {{ .Values.nodePort.enableHealthCheckLoadBalancerIP | quote}}
```

---

Added your suggestion in latest commit